### DISCUSSION TITLE
Diffing animation of adding new chart

### DISCUSSION THREAD

**Discussion by BartNijland91** [patrykandpatrick/vico#350] [Ideas]
Awesome library! I'm working on replacing https://github.com/PhilJay/MPAndroidChart with it, and so far it's been a breeze! 

I'm wondering if it's possible to dynamically add a new chart to `ComposedChartEntryModelProducer` while only animating the newly added Chart. Currently when adding a new one, you have to recreate `ComposedChartEntryModelProducer` and it will animate both charts again.

**Comment by Gowsky** [comment]
Hi @BartNijland91,
It's not possible currently, but this is definitely something we can add in future versions.

**Comment by patrickmichalik** [comment]
Hello! [Vico 1.13.0](https://github.com/patrykandpatrick/vico/releases/tag/v1.13.0) addressed this. You no longer have to recreate a `ComposedChartEntryModelProducer` to add a new data set, and so difference animations behave as requested.
